src/include/rbtree.h

```diff
@@ -2,16 +2,162 @@
 #define RBTREE_H
 
 #include <stdint.h>
-
+/*
+The main functionality of the red-black trees is to provide a fast, 
```

Comment:
Specify the abbreviated form of red-black tree as "rbtree" in the first occurrence, so that "rbtree" can appear in the latter context, making comments more clear.